components/developer/swig/Makefile

```diff
@@ -79,23 +79,17 @@ COMPONENT_TEST_TARGETS = \
         partialcheck-python-test-suite \
         partialcheck-ruby-test-suite
 
-# common targets
-build:		$(BUILD_32_and_64)
-
-install:	$(INSTALL_32_and_64)
-
-test:		$(TEST_32_and_64)
-
-REQUIRED_PACKAGES += developer/java/openjdk7
+# Build dependencies
+PERL_REQUIRED_PACKAGES += runtime/perl
+REQUIRED_PACKAGES += runtime/java/openjdk11
 REQUIRED_PACKAGES += developer/parser/bison
```

Comment:
developer/parser/bison has been removed and replaced by
developer/parser/bison-37
or
developer/parser/bison-38

Reply:
Hmm, this is unfortunat, however developer/parser/bison will install developer/parser/bison-37. I do not feel confortable using a versionedfmri for such an essential package. Otherwise anytime we bump we'll have to change the dependency in each component. Probably developer/parser/bison should always point to the bison-of-the-day.

Reply:
Yes, I know. The problem was that I updated bison 3.8 and later found out that it broke some packages. Thus, I changed it fmri to bison-38 and introduced the old version as bison-37. We had similar situations for automake and some other essential packages, too.

Reply:
OK perfect :) the component as it is does not cause problem and uses bison-37 so let us leave it as it is...